lib/Conversation/CommentForm/index.js

```diff
@@ -58,8 +60,10 @@ class CommentForm extends Component {
     if (e) {
       e.preventDefault();
     }
-    this.props.onSubmit(this.state.inputValue);
-    const inputValue = !this.props.isSubmitting ? '' : this.state.inputValue;
+    this.props.onSubmit(this.removeMentionMarkup());
+    const inputValue = !this.props.isSubmitting
```

Comment:
Could you share some knowledge for how the submitting/saving state should perform? It's always been a gap in my knowledge about Conversations.

Reply:
You know, I cant even remember. I suspect it was something that was previously used, but isn't any more? I can't see anywhere where its set to true. Loath to change it now just incase it somehow breaks it, and its not really hurting anything right now. Maybe worth reviewing on a future refactor? We've thrown around the idea of refactoring conversation a few times 🙂

Reply:
Putting this here too for clarity: A while back we added functionality for a comment submitting state (isSubmitting) if isSubmitting is true we’d display the comment form still populated with text for a little longer with the text area greyed out/disabled and the submit button would become a spinner.

This was either never hooked up and forgotten about (likely) or taken out for other reasons